client/my-sites/stats/stats-redirect/load-stats-page.tsx

```diff
@@ -99,5 +100,10 @@ export default function LoadStatsPage( { children }: AsyncLoadProps ) {
 		return <EnableStatsModuleNotice siteId={ siteId } path={ path } />;
 	}
 
-	return <StatsRedirectFlow>{ children }</StatsRedirectFlow>;
+	return (
+		<StatsRedirectFlow>
+			{ isOdysseyStats && <QuerySitePurchases siteId={ siteId } /> }
```

Comment:
hmmm it seems like the query is already sent by Calypso somewhere, hence the dead loop for Calypso Stats... 😢